packages/jest-circus/src/utils.js

```diff
@@ -110,46 +110,59 @@ const getEachHooksForTest = (
   return result;
 };
 
+const _makeTimeoutMessage = (timeout, isHook) => {
+  const message = `Exceeded timeout of ${timeout}ms for a ${isHook ? 'hook' : 'test'}.`;
```

Comment:
i actually did it first and then realized that the name of the test is already printed right before the error (cause it's going to be under the failing test) 🙂